released/packages/coq-qcert/coq-qcert.1.4.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+name: "coq-qcert"
+version: "1.4.0"
```

Comment:
I think these fields could be removed as they are redundant with the package file name.

Reply:
(I mean `name` and `version`)

Reply:
I've removed `name` and `version` as suggested.

Reply:
Thanks!